xwiki-platform-core/xwiki-platform-legacy/xwiki-platform-legacy-events-hibernate/xwiki-platform-legacy-events-hibernate-api/pom.xml

```diff
@@ -24,22 +24,21 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>org.xwiki.platform</groupId>
-        <artifactId>xwiki-platform-eventstream-stores</artifactId>
+        <artifactId>xwiki-platform-legacy-events-hibernate</artifactId>
         <version>14.5-SNAPSHOT</version>
     </parent>
-    <artifactId>xwiki-platform-eventstream-store-hibernate</artifactId>
-    <name>XWiki Platform - Event Stream - Store - Hibernate</name>
-    <description>Hibernate based events storage</description>
+    <artifactId>xwiki-platform-legacy-events-hibernate-api</artifactId>
+    <name>XWiki Platform - Legacy - Events Hibernate - API</name>
+    <description>Hibernate based events storage and associated legacy APIs</description>
     <properties>
-        <xwiki.jacoco.instructionRatio>0.18</xwiki.jacoco.instructionRatio>
-        <!-- Name to display by the Extension Manager -->
-        <xwiki.extension.name>Hibernate Event Stream Store</xwiki.extension.name>
+        <xwiki.jacoco.instructionRatio>0.20</xwiki.jacoco.instructionRatio>
 
         <xwiki.extension.features>
           <!-- Previous name of the module -->
           org.xwiki.platform:xwiki-platform-eventstream-store
+          org.xwiki.platform:xwiki-platform-legacy-eventstream-store-hibernate
```

Comment:
you meant `xwiki-platform-eventstream-store-hibernate` here no?

Reply:
Yes, fixing.